Detectors/Calibration/include/DetectorsCalibration/MeanVertexData.h

```diff
@@ -0,0 +1,231 @@
+    nbinsY = other.nbinsY;
+    rangeY = other.rangeY;
+    nbinsZ = other.nbinsZ;
+    rangeZ = other.rangeZ;
+    v2BinX = other.v2BinX;
+    v2BinY = other.v2BinY;
+    v2BinZ = other.v2BinZ;
+    histoX = other.histoX;
+    histoY = other.histoY;
+    histoZ = other.histoZ;
+    other.useFit = false;
+    other.nbinsX = 1000;
+    other.rangeX = 10.f;
+    other.nbinsY = 1000;
+    other.rangeY = 10.f;
+    other.nbinsZ = 1000;
+    other.rangeZ = 10.f;
+    other.v2BinX = nbinsX / (2 * rangeX);
+    other.v2BinY = nbinsY / (2 * rangeY);
+    other.v2BinZ = nbinsZ / (2 * rangeZ);
+    other.histoX.clear();
+    other.histoY.clear();
+    other.histoZ.clear();
+}
+
+//_____________________________________________
+MeanVertexData(const MeanVertexData& other) {
+
+    // copy constructor
+    
+    useFit = other.useFit;
+    nbinsX = other.nbinsX;
+    rangeX = other.rangeX;
+    nbinsY = other.nbinsY;
+    rangeY = other.rangeY;
+    nbinsZ = other.nbinsZ;
+    rangeZ = other.rangeZ;
+    v2BinX = other.v2BinX;
+    v2BinY = other.v2BinY;
+    v2BinZ = other.v2BinZ;
+    histoX = other.histoX;
+    histoY = other.histoY;
+    histoZ = other.histoZ;
+}
+
+//_____________________________________________
+MeanVertexData& operator = (MeanVertexData& other) {
+
+    // assignment operator
+    
+    useFit = other.useFit;
+    nbinsX = other.nbinsX;
+    rangeX = other.rangeX;
+    nbinsY = other.nbinsY;
+    rangeY = other.rangeY;
+    nbinsZ = other.nbinsZ;
+    rangeZ = other.rangeZ;
+    v2BinX = other.v2BinX;
+    v2BinY = other.v2BinY;
+    v2BinZ = other.v2BinZ;
+    histoX = other.histoX;
+    histoY = other.histoY;
+    histoZ = other.histoZ;
+    other.useFit = false;
+    other.nbinsX = 1000;
+    other.rangeX = 10.f;
+    other.nbinsY = 1000;
+    other.rangeY = 10.f;
+    other.nbinsZ = 1000;
+    other.rangeZ = 10.f;
+    other.v2BinX = nbinsX / (2 * rangeX);
+    other.v2BinY = nbinsY / (2 * rangeY);
+    other.v2BinZ = nbinsZ / (2 * rangeZ);
+    other.histoX.clear();
+    other.histoY.clear();
+    other.histoZ.clear();
+    return *this;
+}
+
+//_____________________________________________
+MeanVertexData& operator = (MeanVertexData&& other) {
+
+    // move assignment operator
+    
+    useFit = other.useFit;
+    nbinsX = other.nbinsX;
+    rangeX = other.rangeX;
+    nbinsY = other.nbinsY;
+    rangeY = other.rangeY;
+    nbinsZ = other.nbinsZ;
+    rangeZ = other.rangeZ;
+    v2BinX = other.v2BinX;
+    v2BinY = other.v2BinY;
+    v2BinZ = other.v2BinZ;
+    histoX = other.histoX;
+    histoY = other.histoY;
+    histoZ = other.histoZ;
+    other.useFit = false;
+    other.nbinsX = 1000;
+    other.rangeX = 10.f;
+    other.nbinsY = 1000;
+    other.rangeY = 10.f;
+    other.nbinsZ = 1000;
+    other.rangeZ = 10.f;
+    other.v2BinX = nbinsX / (2 * rangeX);
+    other.v2BinY = nbinsY / (2 * rangeY);
+    other.v2BinZ = nbinsZ / (2 * rangeZ);
+    other.histoX.clear();
+    other.histoY.clear();
+    other.histoZ.clear();
```

Comment:
idem. Also, for move c-tor there is no point in resetting the original object.

Reply:
Idem, default used now. If it is ok to rely on that for the std::move, then it is fine with me.